grape-swagger.gemspec

```diff
@@ -15,6 +15,7 @@ Gem::Specification.new do |s|
 
   s.required_ruby_version = '>= 2.4'
   s.add_runtime_dependency 'grape', '>= 0.16.2', '< 1.3.0'
+  s.add_runtime_dependency 'rack', '2.0.8'
```

Comment:
This locks dependency to a specific version of Rack. Definitely don't want to do that as it will render grape-swagger incompatible with most apps. 

Grape 1.3 is compatible with rack 2.1, there's a matrix where you can make it all work. LMK if you need help.

Reply:
Yeap I know, the problem here isn’t grape, it is grape-swagger and I’m on it.

Sorry for circumstances, if you need rack > 2.0.x use version 0.34.1, there rack isn’t frozen.